### ISSUE TITLE
Find (free?) alternative to pingdom

### ISSUE THREAD

**Issue by molant** [webhintio/webhint.io#591]
Starting on February 6 Pingdom will stop offering their free tier. We are currently using Pingdom to track downtime. We need to find an alternative that hopefully allow us to do a bit more than just tracking the homepage. Ideally we should monitor a few endpoints that have given us problems in the pass:

* [ ] Scanner + results
* [ ] Search + results
* [ ] Something else?

**Comment by molant** [comment]
IIRC Azure has something that we could use so I'll probably start looking into that.

**Comment by molant** [comment]
I've added some availability tests to Azure:

![image](https://user-images.githubusercontent.com/606594/50811250-66d98e80-12c2-11e9-966b-97bfa569ee7e.png)

I've added tests for:
* homepage
* scanner
* search results
* scanner results
* 404 page

I was looking into adding a [multi-step test](https://docs.microsoft.com/en-us/azure/azure-monitor/app/monitor-web-app-availability#multi-step-web-tests) but it looks like we need VS 2017 Enterprise to create them.

If something fails the contact email for webhint.io will be notified.

I've also canceled our pingdom account so I'll close this as I don't think there's a lot more to do.
